config/crds/ibmcloud_v1alpha1_binding.yaml

```diff
@@ -43,6 +43,8 @@ spec:
               type: string
             serviceName:
               type: string
+            serviceNamespace:
+              type: string
```

Comment:
@vazirim is this required for issue #21 ? Changes to CRDs might be a little tricky because now we have released the CRDs as v1alpha1 in operator hub and changing the CRD will require creating new CRD versions, new release and new PR to operator hub to release a new version that replaces 0.1.0